lerobot/common/datasets/v2/batch_convert_dataset_v1_to_v2.py

```diff
@@ -0,0 +1,880 @@
+#!/usr/bin/env python
+
+# Copyright 2024 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Note: Since the original Aloha datasets don't use shadow motors, you need to comment those out in
+lerobot/configs/robot/aloha.yaml before running this script.
```

Comment:
What about mobile Aloha?

Reply:
This is only for datasets recorded with our own scripts from v1 in which we haven't used features from mobile Aloha, so should be an issue. We should however add them in the future in the example conversion script to port external datasets.